packages/strapi-utils/lib/__tests__/content-types.test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const { isPrivateAttribute, getPrivateAttributes } = require('../content-types');
+
+const createModel = (privateAttributes = []) => ({
+  options: {
+    privateAttributes,
+  },
+  attributes: {
+    foo: {
+      type: 'string',
+      private: true,
+    },
+    bar: {
+      type: 'number',
+      private: false,
+    },
+    foobar: {
+      type: 'string',
+    },
+  },
+});
+
+const createConfig = (privateAttributes = []) => ({
+  get: jest.fn(() => privateAttributes),
+});
+
+describe('Content-Types', () => {
```

Comment:
You can add a test to check the config.get function is called with the right param.